package.json

```diff
@@ -85,7 +85,7 @@
     "@metamask/eth-sig-util": "^5.0.2",
     "@nomicfoundation/hardhat-toolbox": "^2.0.0",
     "@nomiclabs/hardhat-ethers": "^2.2.1",
-    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client",
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#PP-635/relay-client-tests",
```

Comment:
we need to update the dependency once we merge the [related PR](https://github.com/rsksmart/rif-relay-client/pull/80)